### PR TITLE
Script for coverage when SDL not receive HMI respond with AutoComplet…

### PR DESCRIPTION
**Pull request by Aleksandriys**
…eList
@aarotar @OVikhrov please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua [hunk 1]
```diff
@@ -0,0 +1,59 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua]: @Aleksandriys, remove duplicated "SDL" in requirement summary

File: test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua]: @Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "GENERIC_ERROR") and not <resultCode_received_from _HMI>

File: test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> 
+-- 3. HMI does NOT respond during <DefaultTimeout>
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua]: @Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3. and p4 should be in expected result.

File: test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {"List_1, List_2", "List_1, List_2"}
+      }
+    })
+--hmi side: Default timeout, SDL recieved GENERIC_ERROR
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/016_ATF_SetGlobalProperties_N_AutoCompleteList_HMI_not_Respond_GENERIC_ERROR.lua]: @Aleksandriys, in test is missed step for ensuring that HMI sends nothing, you just expect GENERIC_ERROR to mobile(comment "--hmi side: Default timeout, SDL recieved GENERIC_ERROR" is not correct).
Please add check, that hmi sends nothing

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Pull request merged**